src/adapters/conversant.js

```diff
@@ -73,14 +73,14 @@ var ConversantAdapter = function () {
 
     //build impression array for conversant
     utils._each(bidReqs, function (bid) {
-      var bidfloor = utils.getBidIdParamater('bidfloor', bid.params),
+      var bidfloor = utils.getBidIdParameter('bidfloor', bid.params),
         sizeArrayLength = bid.sizes.length,
         adW = 0,
         adH = 0,
         imp;
 
-      secure = utils.getBidIdParamater('secure', bid.params) ? 1 : secure;
-      siteId = utils.getBidIdParamater('site_id', bid.params);
+      secure = utils.getBidIdParameter('secure', bid.params) ? 1 : secure;
+      siteId = utils.getBidIdParameter('site_id', bid.params);
```

Comment:
Just merged in a PR from Conversant that's causing a conflict here, but all that is needed to resolve is to change this line to add a string concatenation to the end of this line like so:

`siteId = utils.getBidIdParameter('site_id', bid.params) + '';`